src/dune_rules/foreign_rules.ml

```diff
@@ -127,16 +132,25 @@ let build_c
   let use_standard_flags = Dune_project.use_standard_c_and_cxx_flags project in
   let base_flags =
     match kind with
-    | Cxx -> Fdo.cxx_flags ctx
+    | Cxx ->
+       List.concat
+         [ get_env_flags "CXXFLAGS"
```

Comment:
You can use `Env.get` instead here. The `Env.t` value should come from ctx.env.

We should probably also guard this behaviour behind dune lang. cc @rgrinberg wdyt?

Reply:
> Do you use a custom workspace file when building dune packages in debian?

I was not familiar with this notion, so no. The way most dune packages are built in debian is defined [here](https://salsa.debian.org/ocaml-team/dh-ocaml/-/blob/master/debhelper/ocaml_dune.pm)... Sorry, it's Perl, but should be understandable: the interesting parts are the last four functions (`build`, `test`, `install` and `clean`).

Reply:
> Guarding it behind a command line flag is fine.

I've implemented that in a separate commit. I'm wondering if this is the right move, though. The feature could be useful for installing opam packages, too, and there is no way to change the command used there...?

Reply:
> What does "wdyt" mean?
"What do you think"